aspnetcore/fundamentals/http-requests.md

```diff
@@ -15,7 +15,7 @@ By [Glenn Condron](https://github.com/glennc), [Ryan Nowak](https://github.com/r
 An <xref:System.Net.Http.IHttpClientFactory> can be registered and used to configure and create <xref:System.Net.Http.HttpClient> instances in an app. It offers the following benefits:
 
 * Provides a central location for naming and configuring logical `HttpClient` instances. For example, a *github* client can be registered and configured to access [GitHub](https://github.com/). A default client can be registered for other purposes.
-* Codifies the concept of outgoing middleware via delegating handlers in `HttpClient` and provides extensions for Polly-based middleware to take advantage of that.
+* Codifies the concept of outgoing middleware via delegating handlers in `HttpClient` and provides extensions for Polly based middleware to take advantage of that.
```

Comment:
"Polly" isn't an adverb here ... it forms part of a *compound modifier* ... in this case prior to the noun, *middleware*. Therefore, I recommend not making this change here and in the other spots below.

Reply:
done